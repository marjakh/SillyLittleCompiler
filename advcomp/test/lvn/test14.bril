@main {
  a: int = const 1;
  b: int = const 2;
  x: int = add a b;
  x: int = const 4;
  y: int = add a b;
  print y;
}
