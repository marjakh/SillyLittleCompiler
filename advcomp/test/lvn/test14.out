@main {
  variable0: int = const 3;
  print variable0;
}
